test/integration/backward_compatible/map/MapProxyTest.js

```diff
@@ -355,6 +358,7 @@ describe('MapProxyTest', function () {
             });
 
             it('putTransient_with_maxIdle', async function () {
+                TestUtil.markClientVersionAtLeast(this, '4.1');
```

Comment:
I wonder the reason why clients older than 4.1 do not support these tests

Reply:
maxIdle parameter support added in 4.1